conduct.md

```diff
@@ -0,0 +1,13 @@
+Team Python Code of Conduct
+1. Everyone should do their part on time.
+   If you experience problems, ask for help. 
+2. Everyone must be respectful and considerate.
+3. Everyone must communicate in an efficient and transparent manner.
+4. The work will be equally divided between all team members, according to their preferences/strong skills etc. 
+5. Everyone should have at least minimal involvement in each part of the project, even if it is not their area of expertise. 
+6. Meetings.
+Everyone is expected to attend the weekly meetings (date and location TBA)
```

Comment:
If you want these sentences to be on separate rows, you can indent (4 spaces) them and prefix with a `- ` or `* ` to make them subbullets of point 6.  For example:

```markdown
6. Meetings.
    - Everyone is expected to attend the weekly meetings (date and location TBA)
    - In case, when a member cannot make it, they should warn other team members in advance.
```
becomes :


6. Meetings.
    - Everyone is expected to attend the weekly meetings (date and location TBA)
    - In case, when a member cannot make it, they should warn other team members in advance.